packages/sindarian-i18n-cli/src/extractor.ts

```
@@ -0,0 +1,202 @@
+import * as ts from 'typescript'
+import { transformWithTs } from '@formatjs/ts-transformer'
+import { readFile } from 'fs/promises'
+import { glob } from 'glob'
```

ESLint

[error] "glob" should be replaced with an alternative package. In your project, we recommend tinyglobby. Read more here: https://e18e.dev/docs/replacements/glob

(e18e/ban-dependencies)